flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/stream/sql/DagOptimizationTest.xml

```diff
@@ -222,7 +211,7 @@ Sink(name=[`default_catalog`.`default_database`.`appendSink1`], fields=[a, b1],
 +- Reused(reference_id=[2])
 
 Sink(name=[`default_catalog`.`default_database`.`appendSink2`], fields=[a, b1], updateAsRetraction=[false], accMode=[Acc])
-+- Union(all=[true], union=[a, b], updateAsRetraction=[false], accMode=[Acc])
++- Union(all=[true], union=[a1, b1], updateAsRetraction=[false], accMode=[Acc])
```

Comment:
The query is `SELECT a as a1, b as b1 FROM MyTable WHERE a <= 10` so the name is actually correct. The original name is wrong.